File: app/src/main/java/com/van/appwidget/PinningReceiver.java
package com.van.appwidget;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class PinningReceiver extends BroadcastReceiver {

    private static final String TAG = "PinningReceiver";

    @Override
    public void onReceive(Context context, Intent intent) {
        Log.i(TAG, "onReceive : " + intent.toString());
    }
}

File: app/src/main/java/com/van/appwidget/PinningReceiver.kt
package com.van.appwidget

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class PinningReceiver: BroadcastReceiver() {

    companion object {
        private const val TAG = "PinningReceiver"
    }

    override fun onReceive(context: Context, intent: Intent) {
        Log.i(TAG, "onReceive : " + intent.toString())
    }
}